pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -2002,6 +2016,31 @@ func makeBasePod(t testingMode, nodeName, objName, cpu, mem, extended string, po
 	return podWrapper.Obj()
 }
 
+func checkNumNodesOfImageState(nodes map[string]*framework.NodeInfo, imageNames ...string) bool {
```

Comment:
this is too much logic again. Try to do direct comparisons.

Reply:
was this addressed?